src/setup/cortx_deploy

```diff
@@ -54,10 +54,44 @@ function conf_get {
     echo $val
 }
 
+function apply_cluster_config {
+    log $INFO "cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c '$CORTX_CONFSTORE' $FORCE_PROVISIONING"
+    cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c "$CORTX_CONFSTORE" $FORCE_PROVISIONING;
+}
+
+function apply_common_config {
+    log $INFO "cortx_setup config apply -f $(yaml_config $CONFIG_INFO) -c '$CORTX_CONFSTORE' $FORCE_PROVISIONING"
+    cortx_setup config apply -f $(yaml_config $CONFIG_INFO) -c "$CORTX_CONFSTORE" $FORCE_PROVISIONING;
+}
+
+function deploy {
+    # Bootstrap Cluster
+    log $INFO "cortx_setup cluster bootstrap -c '$CORTX_CONFSTORE' $FORCE_PROVISIONING"
+    cortx_setup cluster bootstrap -c "$CORTX_CONFSTORE" $FORCE_PROVISIONING;
+}
+
+function upgrade {
+    #Upgrade cluster
+    log $INFO "cortx_setup cluster upgrade -c '$CORTX_CONFSTORE' $FORCE_PROVISIONING"
+    cortx_setup cluster upgrade -c "$CORTX_CONFSTORE" $FORCE_PROVISIONING;
+}
+
+function validate{
```

Comment:
1. Spacing before {
2. Dont assume global variable. Call this conditionally
```
function validate_conf { 
        log $INFO "cortx_setup config validate -f $(yaml_config $CLUSTER_INFO) -c '$CORTX_CONFSTORE'"
        cortx_setup config validate -f $(yaml_config $CLUSTER_INFO) -c "$CORTX_CONFSTORE" || return 1
        log $INFO "cortx_setup config validate -f $(yaml_config $CONFIG_INFO) -c '$CORTX_CONFSTORE'"
        cortx_setup config validate -f $(yaml_config $CONFIG_INFO) -c "$CORTX_CONFSTORE" || return 1
        return 0
}

# call 
[ ! -z $VALIDATE_CONF ] && validate_conf
```

Reply:
Please change validate_config function to validate_config_file to avoid conflict with this function.

Reply:
Done.